### PR TITLE
GUI Geocode Locator to fix #32

### LINKED ISSUE #32: Geo-Co-ordinate Finder
# Aim

- To find the Latitude and Longitude of a given Address.

# Details

- A GUI which will ask the user to enter their address and in return will give them the Latitude and Longitude of that Address

### PR DESCRIPTION
**Pull request by abirbhattacharya82**
# Description

The issue was to create a GUI Based geo coordinate locator which would give the user the Latitude and Longitude when they provide a particular address.

## Related Issues or Pull Requests

#32 

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

### CODE REVIEW

File: Geo-Cordinate Locator/script.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from tkinter import *
```
Comment (avinashkranjan) [inline Geo-Cordinate Locator/script.py:1]: Kindly add packages names in comment 
``` 
Package Included
1. pip install tkinter
2. pip install geopy
```

Or Better alternative use `requirements.txt` mentioning all the packages with version..

File: Geo-Cordinate Locator/script.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+b1=Button(window,text="Locate Me!",width=40,command=f)
+b1.grid(row=1,column=0,columnspan=2,)
+
+l2=Label(window,text="Latitude")
+l2.grid(row=2,column=0)
+
+l3=Label(window,text="Longitude")
+l3.grid(row=3,column=0)
+
```
Comment (avinashkranjan) [inline Geo-Cordinate Locator/script.py:34]: Add a **copy button** to copy the latitude and longitude..for better **User-Convenience**

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Rest Look good to me..Just make the asked changes and it'll be ready for Merge

File: Geo-Cordinate Locator/script.py (hunk 2 above)
Comment (abirbhattacharya82) [reply Geo-Cordinate Locator/script.py:34]: can you be a little more specific. Like where do you want to copy the geo- cordinates?

File: Geo-Cordinate Locator/Requirements.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+The packages used are:
+1) tkinter
+2) geopy
+
+If geopy is not installed, open command prompt and use the following command:
+pip install geopy
+I hope it resolves any kind of problems that may arise.
+Also make sure your machine is connected to Internet while you are executing this program.
```
Comment (avinashkranjan) [inline Geo-Cordinate Locator/Requirements.txt:8]: Your don't write requirements like this..
1. Change **Requirements.txt** to **requirements.txt**
2. Here's how you write **requirements.txt**
```
tkinter==8.6.10
geopy==2.0.0
```
So, you can install them all the packages used at once `pip install -r requirements.txt`

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Make the Asked changes

File: Geo-Cordinate Locator/script.py (hunk 2 above)
Comment (avinashkranjan) [reply Geo-Cordinate Locator/script.py:34]: To the clipboard..So I paste where ever I want.

File: Geo-Cordinate Locator/Requirements.txt (hunk 3 above)
Comment (abirbhattacharya82) [reply Geo-Cordinate Locator/Requirements.txt]: check if its alright or not.

File: Geo-Cordinate Locator/Requirements.txt [hunk 4]
```diff
@@ -1,9 +1,3 @@
...
-2) geopy
-
-If geopy is not installed, open command prompt and use the following command:
-pip install geopy
-I hope it resolves any kind of problems that may arise.
-Also make sure your machine is connected to Internet while you are executing this program.
-[All above mentioned packages work well for Python3]
+tkinter==8.6.10
+geopy==2.0.0
+So, you can install them all the packages used at once pip install -r requirements.txt
```
Comment (avinashkranjan) [inline Geo-Cordinate Locator/Requirements.txt]: Remove this line

**Review by avinashkranjan — CHANGES_REQUESTED** [review]

File: Geo-Cordinate Locator/Requirements.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (avinashkranjan) [inline Geo-Cordinate Locator/Requirements.txt]: Remove 
`So, you can install them all the packages used at once pip install -r requirements.txt` this line I wrote that for you

**Review by avinashkranjan — CHANGES_REQUESTED** [review]

File: Geo-Cordinate Locator/Requirements.txt (hunk 5 above)
Comment (abirbhattacharya82) [reply Geo-Cordinate Locator/Requirements.txt]: oh achchaa. Got it

Comment (abirbhattacharya82) [reply Geo-Cordinate Locator/Requirements.txt]: done

**Review by avinashkranjan — APPROVED** [review]
Perfect..and thats how you write a `requirements.txt` file

File: Geo-Cordinate Locator/script.py (hunk 2 above)
Comment (avinashkranjan) [reply Geo-Cordinate Locator/script.py:34]: @abirbhattacharya82 Kindly look into these..

Comment (abirbhattacharya82) [reply Geo-Cordinate Locator/script.py:34]: alright I am looking into  it

**Review by avinashkranjan — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**